src/Microsoft.DocAsCode.Dfm.VscPreview/DfmProcess.cs

```diff
@@ -63,11 +66,36 @@ private static void ReceiveContent()
             Console.Write('\a');
         }
 
+        private static void JsonMarkupReceiveContent()
+        {
+            // a simple protocol(get String According to the numOfRow and connect them)
+            string numStr = Console.ReadLine();
+            int numOfRow = Convert.ToInt32(numStr);
+            StringBuilder markdownContent = new StringBuilder();
+            for (int i = 0; i < numOfRow; i++)
+            {
+                markdownContent.AppendLine(Console.ReadLine());
+            }
+
+            var result = JsonMarkup(markdownContent.ToString());
+
+            // append with customized endCode
+            Console.Write(result);
+            Console.Write('\a');
```

Comment:
for the common code with DfmMarkupReceiveContent, why not extract a common method